tests/MongoDB.Driver.Core.Tests/Specifications/connection-string/ConnectionStringTestRunner.cs

```diff
@@ -0,0 +1,354 @@
+/* Copyright 2015-present MongoDB Inc.
```

Comment:
It sucks that github pull request does not realize that this file was just renamed.

By treating it as a new files we can't see the diffs in the pull request.